CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 This document assumes you have followed the [Apache Geode Code contribution instructions](https://cwiki.apache.org/confluence/display/GEODE/Code+contributions)
 
 ## Building the code
-    see BUILDING.md
+see BUILDING.md
```

Comment:
if you use `[BUILDING.md](BUILDING.md)` then a nice link appears for clicking through